command/node_drain.go

```diff
@@ -271,16 +271,20 @@ func (c *NodeDrainCommand) Run(args []string) int {
 		return 1
 	}
 
-	if enable {
-		c.Ui.Output(fmt.Sprintf("Node %q drain strategy set", node.ID))
-	} else {
-		c.Ui.Output(fmt.Sprintf("Node %q drain strategy unset", node.ID))
+	if !enable || detach {
+		if enable {
+			c.Ui.Output(fmt.Sprintf("Node %q drain strategy set", node.ID))
+		} else {
+			c.Ui.Output(fmt.Sprintf("Node %q drain strategy unset", node.ID))
+		}
 	}
 
 	if enable && !detach {
+		c.Ui.Output(fmt.Sprintf("%s Node %q drain strategy set", formatTime(time.Now()), node.ID))
+		c.Ui.Output(" (Ctrl-C to stop monitoring: will not cancel the node drain)")
```

Comment:
Any reason for the space in front? Also What about putting this as the first line?